annotations/src/main/resources/org/leandreck/endpoints/templates/typescript/apimodule.ftl

```diff
@@ -16,16 +16,30 @@
     limitations under the License.
 
 -->
-import { NgModule } from '@angular/core';
+import { NgModule, ModuleWithProviders, Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
 <#list endpoints as service>
 import { ${service.serviceName} } from './${service.serviceName?lower_case}.generated';
 </#list>
 
-@NgModule({
-    providers: [
-        <#list endpoints as service>
-        ${service.serviceName}<#sep>,</#sep>
-        </#list>
-    ],
-})
-export class APIModule { }
\ No newline at end of file
+@Injectable()
+export interface ServiceConfig {
```

Comment:
@mkowalzik What do you think of this signature for ServiceConfig?

Reply:
I think it is quite clear what it means, perhaps we should add some TypeDoc?